usr/share/rear/verify/BORG/default/400_check_archive_access.sh

```diff
@@ -0,0 +1,45 @@
+# Query Borg server for repository information
+# and store it to BORGBACKUP_ARCHIVE_CACHE.
+# This should avoid repeatedly querying Borg server, which could be slow.
+
+borg_archive_cache_create && return
+
+LogPrint "Failed to list Borg archive."
+LogPrint "If you decide to continue, ReaR will partition your disks, but most probably will NOT be able to restore your data!"
+Log "Command \"borg list $BORGBACKUP_OPT_REMOTE_PATH ${borg_dst_dev}${BORGBACKUP_REPO}\" returned: "
+borg_list
+
+rear_workflow="rear $WORKFLOW"
+choices[0]="View '$rear_workflow' log file ($RUNTIME_LOGFILE)"
+choices[1]="Use Relax-and-Recover shell and return back to here"
+choices[2]="Continue '$rear_workflow'"
+choices[3]="Abort '$rear_workflow'"
+
+prompt="Do you want to continue?"
+choice=""
+wilful_input=""
+
+while true ; do
+    # Read user input.
+    choice="$( UserInput -I BORGBACKUP_CONTINUE_WITH_RECOVER -p "$prompt" -D "${choices[3]}" "${choices[@]}" )" && wilful_input="yes" || wilful_input="no"
+
+    case "$choice" in
+        (${choices[0]})
+            # Run 'less' with the original STDIN STDOUT and STDERR when 'rear' was launched by the user:
+            less $RUNTIME_LOGFILE 0<&6 1>&7 2>&8
+        ;;
+        (${choices[1]})
+            # rear_shell runs 'bash' with the original STDIN STDOUT and STDERR when 'rear' was launched by the user:
+            rear_shell "" "$rear_shell_history"
+        ;;
+        (${choices[2]})
+            # Continue recovery in migration mode:
```

Comment:
The `# Continue recovery in migration mode:` comment
is likely not right in this case here (probably via copy&paste)
and should be in this case here just something like
`# Continue recovery:`

Reply:
Right!  :thumbsup:

Reply:
Of course!
I know my code and see when you "steal" it ;-)
Of course you are welcome and I appreciate it when my code gets reused.